server/api/receipt_controller.py

```diff
@@ -58,7 +56,7 @@ def get_all_receipts(year, month):
         receipts = Receipt.query.all()
         # Receipt.query.all() returns empty list if there are no records
         if receipts == []:
-            return receipts
+            return jsonify({'receipts': receipts})
```

Comment:
I actually ended up removing this line in my branch cause the function returns total_amount = 0 and posts =[], so it was redundant to have this line. I'll be pushing my code onto the dev branch soon so you'll get the change later :)

Reply:
thanks. sounds good!